bitnami/odoo/values.yaml

```diff
@@ -369,7 +368,6 @@ terminationGracePeriodSeconds: ""
 ##
 updateStrategy:
   type: RollingUpdate
-  rollingUpdate: {}
```

Comment:
```suggestion
  rollingUpdate: null
```

Reply:
Hey @juan131! TIL! Didn't know it will be ignored if `null`. Nice!